tests/sentry/backup/test_models.py

```diff
@@ -556,7 +567,7 @@ def test_user_ip(self):
         )
         return self.import_export_then_validate()
 
-    @targets_models(UserRole, UserRoleUser)
+    @targets(mark(UserRole, UserRoleUser))
```

Comment:
Do we ever call `targets` without calling `mark`? If not, then why can't the logic in `mark` be folded into `targets`?

Reply:
Because the decorator only runs when the tests are executed. For the pattern as its currently set up to work, we need the tests to be "marked" at module load time, so that when I do for example `pytest .../test_coverage.py` it passes without issue.

Another approach would be to have a final `test_coverage` or similar method in `test_models.py`, but this is brittle because it depends on test ordering and fails when a subset of tests are run. The current solution works statically, which is a better DX, especially because these tests aren't super fast.

Reply:
Okay, makes sense. Doc suggestion [above](https://github.com/getsentry/sentry/pull/53295/files#r1271018530), in that case.